ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/services/ActiveContractsServiceAuthorization.scala

```diff
@@ -15,23 +15,21 @@ import com.digitalasset.platform.api.grpc.GrpcApiService
 import com.digitalasset.platform.server.api.ProxyCloseable
 import io.grpc.ServerServiceDefinition
 import io.grpc.stub.StreamObserver
-import org.slf4j.{Logger, LoggerFactory}
 
-class ActiveContractsServiceAuthorization(
+final class ActiveContractsServiceAuthorization(
     protected val service: ActiveContractsService with AutoCloseable,
-    protected val authService: AuthService)
+    private val authorizer: Authorizer,
+    private val authService: AuthService)
```

Comment:
I have added the field when I thought the wrappers will need to re-decode the claims to support token revocation during streaming responses.

It can be removed now, the `AuthService` now only communicates with the wrappers by placing immutable claims in `Context`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3286/commits/0799c10dc275376fa91a59be992692f4ccaa6944